src/models/log-v4.js

```diff
@@ -33,10 +39,10 @@ async function displayLogs (params) {
 
   logStream
     .on('open', (event) => {
-      Logger.debug(`stream opened! ${JSON.stringify({ appId, filter, deploymentId })}`);
+      Logger.debug(colors.blue(`Logs stream (open) ${JSON.stringify({ appId, filter, deploymentId })}`));
```

Comment:
Is-it it possible to set this log not if debug (to answer the "Waiting for application logs…" one?), as the delay between the print of first log line can be long on some applications